src/js/templates/maps/layer-item.html

```diff
@@ -1,12 +1,14 @@
 <button class="<%= classes.visibilityToggle %> map-view__button" title="Toggle layer visibility">
   <i class="icon-eye-open"></i>
 </button>
-<span class="<%= classes.label %>">
+
+<span class="<%= classes.label %> layer-item__label-wrapper">
```

Comment:
Rather than adding a new `layer-item__label-wrapper` class here, it would be better to modify the class that already is set on the label (whatever `classes.label` is set to). This keeps the CSS easier to read and understand. It also allows to keep using dynamic css classes here which makes the view easier to update moving forward.